notebooks/logging-examples/logging-plots.ipynb

```diff
@@ -0,0 +1,330 @@
+      "text/plain": [
+       "<PIL.PngImagePlugin.PngImageFile image mode=RGBA size=700x500 at 0x166F44730>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "display(scatter_plot_image)\n",
```

Comment:
can you put the two display calls each in their own cell so the output doesn't scroll?